README.md

```diff
@@ -3,6 +3,17 @@
 This README would normally document whatever steps are necessary to get the
 application up and running.
 
+For the truck app
+Simply run: `rails s -p 3000`
+
+For the train api
+Go to the train api https://github.com/mmmmaximus/rails-train-api
```

Comment:
```suggestion
Go to the [train api](https://github.com/mmmmaximus/rails-train-api)
```

this will make a markdown link [like this](www.google.com)